### PR TITLE
debug: use latest CRI-O

### PR DESCRIPTION
**Pull request by liubin**

### CODE REVIEW

File: .ci/configure_crio_for_kata.sh [hunk 1]
```diff
@@ -16,9 +16,13 @@ crio_config_dir="/etc/crio/crio.conf.d"
 runc_flag="\/usr\/local\/bin\/crio-runc"
 kata_flag="\/usr\/local\/bin\/containerd-shim-kata-v2"
 
+echo "crio --version!!!!!!!!!!!!"
```
Comment (jodh-intel) [inline .ci/configure_crio_for_kata.sh:19]: I think you missed an exclamation mark there! :smile: But seriously, all the exclamation marks might cause you problems due to the special meaning of `!!`.

Comment (liubin) [reply .ci/configure_crio_for_kata.sh:19]: Sorry, this is only a test :-)

**Pull request closed without merging**